test_vm/tests/test_vm_test.rs

```
@@ -0,0 +1,28 @@
+use fil_actors_runtime::test_utils::{make_builtin, ACCOUNT_ACTOR_CODE_ID, PAYCH_ACTOR_CODE_ID};
+use fvm_shared::address::Address;
+use fvm_shared::blockstore::MemoryBlockstore;
+use fvm_shared::econ::TokenAmount;
+use test_vm::{actor, VM};
+
+#[test]
+fn state_control() {
+    let store = MemoryBlockstore::new();
+    let mut v = VM::new(&store);
+
+    // set actor
+    let a1 = actor(*ACCOUNT_ACTOR_CODE_ID, make_builtin(b"a1-head"), 42, TokenAmount::from(10u8));
+    v.set_actor(Address::new_id(1000), a1.clone()).unwrap();
```

Maybe, for testing, you wanna just panic rather than return `Result`?